### DISCUSSION TITLE
Need help displaying Users/relationships in CP

### DISCUSSION THREAD

**Discussion by jorisvan-ons** [statamic-rad-pack/runway#232] [Help]
I have two fields defined as follows:
```
 [
      'handle' => 'user',
      'field' => [
          'display' => 'User',
          'type' => 'users',
      ],
  ],
```
and 
```
 [
     'handle' => 'subscription_id',
     'field' => [
         'resource' => 'subscription',
         'display' => 'Subscription',
         'type' => 'belongs_to'
     ],
 ],
```
The user fields shows this in the CP:
```
[[ { "id": 2, "title": "Norberto O'Kon", "edit_url": "https://{{ URL }}/cp/users/2/edit", "published": null } ]](https://{{ URL }}/cp/runway/subscription/98a53939-3715-42e2-82d9-8b7890c04233)
```

and the subscription is showed like this:
```
[[ { "id": 1, "title": [], "edit_url": "https://{{ URL }}/cp/runway/subscription/98a53939-3715-42e2-82d9-8b7890c04233" } ]](https://{{ URL }}/cp/runway/payment/1)
```

I've been searching the documentation and source code but I cannot figure out how to configure that it only shows the title attribute (and how to specify what the title should be). Can anyone help here?

**Comment by duncanmcclean** [comment]
Where are you seeing them like that in the Control Panel? On the listing table or on the publish form?

**Reply by jorisvan-ons** [reply]
On the listing table

**Comment by duncanmcclean** [comment]
I've converted this Discussion into an Issue as it sounds more like a bug, see https://github.com/duncanmcclean/runway/issues/234
